lib/rules/no-compare-relation-boolean.js

```diff
@@ -86,6 +86,9 @@ module.exports = {
 
         function checkAssertArguments(callExprNode) {
             const args = [...callExprNode.arguments];
+            if (args.length <= 2) {
```

Comment:
I assume this should be less than 2 rather than less than or equal to 2. Let me know if I'm wrong though 😄